aten/src/ATen/native/native_functions.yaml

```diff
@@ -843,6 +843,22 @@
   dispatch:
     CPU, CUDA: bitwise_not_out
 
+- func: copysign.Tensor(Tensor self, Tensor other) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+  dispatch:
+    CPU, CUDA: copysign
+
+- func: copysign.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
+    CPU, CUDA: copysign_out
+
+- func: copysign.Scalar(Tensor self, Scalar other) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+  dispatch:
+    CPU, CUDA: copysign
```

Comment:
@peterjc123 We are getting a really weird windows linker error on this PR. As the resident windows expert, could you help us take a look please? The failure looks like 
 https://gist.github.com/zou3519/9c557a95d467b4ff17f497abc7948507 -- have you seen anything like that before? Full circle ci logs [are here](https://app.circleci.com/pipelines/github/pytorch/pytorch/229210/workflows/8b49256e-dc8f-476d-8e6f-d9152e5ce62d/jobs/8346245)

Reply:
Are you missing some definitions of the functions here? You can take a look at `bernoulli`.

Reply:
@peterjc123 we discovered something interesting. https://github.com/pytorch/pytorch/pull/46619 (a copy of this PR) fails the windows build, but renaming all of the at::copysign to at::notcopysign (https://github.com/pytorch/pytorch/pull/46620) succeeds building. So it looks like the problem is that we can't name something at::copysign. Do you have any idea why this could be the case?

Reply:
We figured it out: it looks like there's a `#define copysign _copysign` in some python code: https://github.com/python/cpython/blob/c60394c7fc9cc09b16e9675a3eeb5844b6d8523f/PC/pyconfig.h#L196